src/lumigo_tracer/spans_container.py

```diff
@@ -327,3 +326,7 @@ def is_activated():
 
 def _get_envs_for_span(has_error: bool = False) -> str:
     return lumigo_dumps(dict(os.environ), Configuration.get_max_entry_size(has_error))
+
+
+def get_max_possible_size() -> int:
```

Comment:
this should be in utils IMO

Reply:
maybe a better name could be `get_size_upper_bound`?

Reply:
It is lumigo_utils.py :)